README.md

```diff
@@ -65,6 +65,26 @@ to build a cohesive PaaS. It is not a turn-key PaaS solution.
 See the [getting started](http://paasta.readthedocs.io/en/latest/installation/getting_started.html)
 documentation for how to deploy PaaSTA.
 
+## Debugging PaaSTA
+
+You can debug PaaSTA by first running ``make vscode_settings``. This make target will add debugging settings in VS Code in launch.json file for running tests, the PaaSTA API, and the following PaaSTA CLI commands:
+
+ * logs
+ * validate
+ * status
+ * rollback
+ * mark-for-deployment
+
+ In order to run the debugger with the PaaSTA CLI commands, you can go to `Run and Debug` tab in VS Code and select the CLI command you want to run the debugger with, then press on `start debugging` button. The same follows for running PaaSTA tests except you would have to select `tox test`.
+
+To run the debugger with the PaaSTA API, please follow the steps below:
+
+1. The first time you run the debugger on PaaSTA API, you will need to create the `etc_paasta_for_development` directory. To do so, go to `Run and Debug` tab in VS Code and select `paasta API` then press on `start debugging` button. Then stop the debugger.
+2. Copy the content of this [Config setup script](https://fluffy.yelpcorp.com/i/8jtPSNxsnhktqnmN6DJTWBHkVq7n1K1n.html) to your `etc_paasta_for_development` directory, e.g. setup_devapi_cluster.sh
```

Comment:
we should probably not put link to the internal fluffly on public github, instead of that update the Internal Wiki on it